From: Ichthyostega <prg@ichthyostega.de>
Date: Thu, 10 Apr 2025 02:32:33 +0200
Subject: lower CMake minimum requirement

Yoshimi now starts to require "modern CMake" --
Ubuntu/Bionic provides 3.10, which hopefully is "modern" enough...

Moreover we can attempt to revert use of new constructs.
---
 src/CMakeLists.txt | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 96fb94e..bd39690 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -18,8 +18,8 @@
 #   You should have received a copy of the GNU General Public License
 #   along with yoshimi.  If not, see <http://www.gnu.org/licenses/>.
 
-cmake_minimum_required (VERSION 3.12)
-cmake_policy (VERSION 3.12)
+cmake_minimum_required (VERSION 3.5)
+cmake_policy (VERSION 3.5)
 cmake_policy (SET CMP0004 NEW)
 if (POLICY CMP0046)
     cmake_policy (SET CMP0046 NEW)
@@ -463,7 +463,7 @@ set(ExternLibraries
     ${FLTK_LIBRARIES}
     ${MXML_LIBRARIES}
     ${ALSA_LIBRARIES}
-    ${JACK_LINK_LIBRARIES}
+    ${JACK_LIBRARIES}
     ${LIBSNDFILE_LIBRARIES}
     ${FFTW3F_LIBRARIES}
     ${LIBCAIRO_LIBRARIES}
